_plugins/zzz_contribute.md

```diff
@@ -12,12 +12,12 @@ info:
   _info: Learn how to contribute your own plugin.
 ---
 
-The plugins shown here are only a smaller selection of those available.  You can find more themes on [our Wiki][].
+Have you made your own plugin and you'd like to see it featured here?  You're in luck!  We're always interested in new submissions.
 
-You can even [contribute][] your own plugin to the community and we'll feature it here.  While you can freely add your plugin to [our Wiki][], if you'd also like to be included on this page, you can follow the simple instructions in our [Submission Guidelines][].
+Our entire website is featured on [GitHub][gh-pages].  Submitting a new item for this gallery is as easy as creating a [Pull Request][].  Head over to our [Submission Guidelines][], where you can learn more about how to structure and format your submission.
 
-We appreciate any and all contributions, whether you've created something new or restored an obsolete plugin, we'd love to see it!
+We appreciate any and all contributions, whether you've created something new or restored an obsolete plugin, we'd love to see it!  And most importantly, thank you for your interest in Pico!  We're a small project, and we couldn't get by without the support of our Community.
 
-[contribute]: {{ site.github.url }}/docs/#contribute
-[our wiki]: {{ site.gh_project_url }}/wiki/Pico-Plugins
+[gh-pages]: {{ site.gh_project_url }}/tree/gh-pages
+[Pull Request]: {{ site.gh_project_url }}/compare
```

Comment:
This should be a link to a help page or something, see https://github.com/picocms/Pico/pull/358#discussion_r80362323